src/ast.mli

```diff
@@ -76,18 +70,37 @@ and stmt =
   | Break
   | Continue
 
-type func = {
-  name : string;
-  return_t : types;
-  formals : formal list;
-  body : stmt list;
+
+type message = {
+    name: string;
+    formals: formal list;
+}
```

Comment:
per offline convo, `sender` cannot be determined at compile-time, so it won't be included in the `message` type.